frontend/src/store.js

```diff
@@ -43,7 +43,7 @@ const UIStore = new Store({
   sectorOptions: sectorOptions,
   highlight: false,
   disclaimer: null,
-  loading: true,
```

Comment:
so far I don't understand why we prefer to use `isDataFetched=false` vs `loading: true` or  `isDataFetched=true` vs `loading: false`

Reply:
so before, I and @punchagan have a discussion on Zulip about this flag,
then, we decided to have a new flag `isDataFetched` to avoid confusion about `loading` a page or waiting for all `UIStore` data fetching to be done.

Reply:
It's fine to use `loading` too, I wasn't keen on a name change, as long as there is only one flag.

Reply:
so for this case, we will use `loading` instead?